java-client/example-utilities/src/main/java/io/deephaven/client/examples/ConnectOptions.java

```diff
@@ -36,18 +38,26 @@ public static ManagedChannel open(ConnectOptions options) {
 
     @Option(names = {"--max-inbound-message-size"}, description = "The maximum inbound message size, " +
             "defaults to 100MB")
-    int maxInboundMessageSize;
+    Integer maxInboundMessageSize;
```

Comment:
I prefer to use boxed here so I can get proper null, instead of relying on 0 - it's not that the user is explicitly nulling it out, it's that they didn't set it explicitly in their CLI params, so we can inherit the defaults (defined by the ClientConfig).

Reply:
As I recall, 0 is illegal, so it must explicitly be set (to 100MB as the default or some other value manually)

Reply:
The previous code did this:

```java
       if (maxInboundMessageSize == 0) {
            builder.maxInboundMessageSize(100 * 1024 * 1024); // 100MB default message size
        } else {
            builder.maxInboundMessageSize(maxInboundMessageSize);
        }
```
Now it does:
```java
        if (maxInboundMessageSize != null) {
            builder.maxInboundMessageSize(100 * 1024 * 1024); // 100MB default message size
        } else {
            builder.maxInboundMessageSize(maxInboundMessageSize);
            builder.maxInboundMessageSize(maxInboundMessageSize);
        }
```

Reply:
On on the object being built:
```java
    @Default
    public int maxInboundMessageSize() {
        return DEFAULT_MAX_INBOUND_MESSAGE_SIZE;
    }
```